src/test/java/com/github/stickerifier/stickerify/media/MediaHelperTest.java

```diff
@@ -16,6 +16,7 @@
 import com.github.stickerifier.stickerify.ResourceHelper;
 import com.github.stickerifier.stickerify.junit.ClearTempFiles;
 import com.github.stickerifier.stickerify.telegram.exception.TelegramApiException;
+import com.pengrad.telegrambot.model.Animation;
```

Comment:
This import is not needed, please delete it

Reply:
Done.